compose/docker-compose.yml

```diff
@@ -1,71 +1,69 @@
 version: "3.4"
 
 services:
+
   netmaker:
     container_name: netmaker
-    image: gravitl/netmaker:REPLACE_SERVER_IMAGE_TAG
+    image: gravitl/netmaker:$SERVER_IMAGE_TAG
```

Comment:
should $SERVER_IMAGE_TAG be wrapped in brackets like the other vars? e.g. ${SERVER_IMAGE_TAG}

Reply:
Both are supported ([docs](https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution)). Ive added a TODO for error handling to NET-155, these minor differences can be handled there as well.